site/docs/2.0/content/font.md

```diff
@@ -19,7 +19,7 @@ employees through a license with Adobe Typekit.
 
 ### Reference link
 
-Arizona Bootstrap uses a Creative Cloud Library with shared governance to
+Arizona Bootstrap uses Adobe Typekit with a centrally managed netid to
```

Comment:
Why not shared governance?

Reply:
Arizona Bootstrap uses a centrally managed Adobe TypeKit account to

Reply:
Arizona Bootstrap uses an Adobe Typekit account with shared governance to
manage our font reference links within Arizona Bootstrap.
?

Reply:
Sounds good.